From c55c0cb5d4885c0e12545fef3946990184e6e5e9 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Sat, 11 Mar 2017 20:23:10 +0800
Subject: [PATCH 1577/5242] MLK-14471-2 arm: dts: imx7ulp: change pad setting
 for touch

commit  7f5857570157310e067634c7df2111fe2379c178 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set OBE for touch RST pin, set IBE for touch RST.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 5f2a332..4e34092 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -300,8 +300,8 @@
 
 		pinctrl_focaltech: focaltechgrp {
 			fsl,pins = <
-				IMX7ULP_PAD_PTF0__PTF0		0x20100
-				IMX7ULP_PAD_PTF1__PTF1		0x20100
+				IMX7ULP_PAD_PTF0__PTF0		0x10143
+				IMX7ULP_PAD_PTF1__PTF1		0x20143
 			>;
 		};
 
-- 
1.7.9.5

